data_pipelines/utils/flood/config.py

```diff
@@ -0,0 +1,26 @@
+TIMEDELTA = 1
+PYTHON_PREFIX = "/home/giltinde"
+GLOFAS_API_URL = "https://cds.climate.copernicus.eu/api/v2"
+UPSTREAM_URL = "https://confluence.ecmwf.int/download/attachments/242067380/uparea_glofas_v4_0.nc?version=2&modificationDate=1668604690076&api=v2"
+S3_OPEN_EPI_PATH = "mnt/openepi-storage"
+OPENEPI_BASE_PATH = PYTHON_PREFIX + "/" + S3_OPEN_EPI_PATH
```

Comment:
We should remove the variables related to local paths, ie. `PYTHON_PREFIX`, `S3_OPEN_EPI_PATH` and `OPENEPI_BASE_PATH`. Places where the `OPENEPI_BASE_PATH` variable is used can be replaced with `settings.base_data_path` from the `data_pipelines.settings` module.

Reply:
Yup!